build.gradle

```diff
@@ -1,12 +1,12 @@
 ext {
-  kotlinVersion = '1.0.5'
+  kotlinVersion = '1.0.6'
 
   compileSdkVersion = 25
   buildToolsVersion = '25.0.2'
 
   dep = [
-      androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta2',
-      apolloPlugin: 'com.apollographql.android:gradle-plugin:0.0.1-SNAPSHOT',
+      androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta3',
```

Comment:
The sample actually wont build with 2.2.3 because of some weird manifest merger reason. I'll just keep it on the beta for now